### PR TITLE
Fix bulkload failed bug

### PR DESCRIPTION
**Pull request by yhmo**
Signed-off-by: yhmo <yihua.mo@zilliz.com>

#19660

### CODE REVIEW

File: internal/util/importutil/import_wrapper.go [hunk 1]
```diff
@@ -186,7 +186,7 @@ func (p *ImportWrapper) fileValidation(filePaths []string, rowBased bool) error
 		size, err := p.chunkManager.Size(context.TODO(), filePath)
 		if err != nil {
 			log.Error("import wrapper: failed to get file size", zap.String("filePath", filePath), zap.Any("err", err))
-			return errors.New("failed to ")
+			return errors.New("failed to get file size of " + filePath)
```
Comment (soothing-rain) [inline internal/util/importutil/import_wrapper.go:189]: Prefer fmt.Errorf() instead.

File: internal/datanode/data_node.go [hunk 2]
```diff
@@ -1169,6 +1169,12 @@ func importFlushReqFunc(node *DataNode, req *datapb.ImportTaskRequest, res *root
 			rowNum = field.RowNum()
 			break
 		}
+
+		if rowNum <= 0 {
+			log.Info("fields data is empty, no need to generate segment")
```
Comment (soothing-rain) [inline internal/datanode/data_node.go:1174]: Need to be more verbose, consider including task ID, shard ID, # of channels, channel name, etc.

File: internal/util/importutil/import_wrapper.go (hunk 1 above)
Comment (yhmo) [reply internal/util/importutil/import_wrapper.go:189]: There are several places use errors.New(). Change all of them can conflict with #19671. I will refine them in the pr #19671

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**